src/scss/_grid.scss

```diff
@@ -1,26 +1,95 @@
-.grid {
-  @extend .clearfix;
+// .continaer {
+//   @extend .clearfix;
+//
+//   $gutter-width: 1em;
+//
+//   [class*="col-"] {
+//     float: left;
+//     box-sizing: border-box;
+//     padding: 0 $gutter-width;
+//   }
 
-  $gutter-width: 1em;
+  // Helper classes e.g. `.col-1-3` or `.pad-3-12`
+//   @each $units in 2, 3, 4, 6, 12, 16, 18, 24, 32 {
+//     @for $span from 1 through $units {
+//       .col-#{$span}-#{$units} {
+//         width: 100% * ($span/$units);
+//       }
+//
+//       @if ( $span < $units ){
+//         .pad-#{$span}-#{$units} {
+//           margin-left: 100% * ($span/$units);
+//         }
+//       }
+//     } // END @for $span
+//   } // END @each $units
+// }
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: lightgrey;
+
+  h1 {
+    text-align: center;
+  }
```

Comment:
These `body` and `h1` styles really don't belong in `_grid.scss`... This file should _just_ contain the styles and classes necessary for the CSS grid... as if it was a feature!